deploy/testnet/revert.sh

```diff
@@ -16,7 +16,7 @@ source ./config.env
 
 container_name="honest_worker"
 chainspec="./chainspec.json"
-rpc_api_endpoint="http://127.0.0.1:9944"
+#rpc_api_endpoint="http://127.0.0.1:9944"
```

Comment:
It looks like this variable is being used here. I remember there was an unused variable error and it seems strange.

Reply:
Just ensure this script do not call another one where this variable is expected to be set, and then remove it entirely. Also you can add `set -u` to the scripts to fail any attempts to use undefined variables.

Reply:
Done